src/components/Account/AccountModule.js

```diff
@@ -40,21 +41,21 @@ function AccountModule({ compact }) {
     resetConnection,
     switchingNetworks,
   } = useWallet()
+
+  const [isSupportedNetwork, setIsSupportedNetwork] = useState(true)
+
   const [opened, setOpened] = useState(false)
   const [activatingDelayed, setActivatingDelayed] = useState(false)
 
-  const toggle = useCallback(() => setOpened(opened => !opened), [])
+  const toggle = useCallback(() => setOpened((opened) => !opened), [])
 
-  const activate = useCallback(
-    async providerId => {
-      try {
-        await connect(providerId)
-      } catch (error) {
-        console.log('error ', error)
-      }
-    },
-    [connect]
-  )
+  const activate = async (providerId) => {
+    try {
+      await connect(providerId)
+    } catch (error) {
+      console.log('error ', error)
```

Comment:
That need be logged? if yes, maybe console.error()?

Reply:
I am still checking this. But yes it should not be a console log there 👌